src/components/FooterPage/Footer.scss

```diff
@@ -0,0 +1,41 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0px;
+    padding: 0px;
+    overflow-x: hidden;
+}
+
+
+.footer_bg {
+
+
+    background-color: #59BDEE !important;
+}
+
+
+.copyright {
+    font-size: 10px;
+    color: rgb(89, 129, 150) !important;
+    text-decoration: none;
+    background-color: transparent;
+}
+
+
+a {
+    color: rgb(89, 129, 150) !important;
```

Comment:
the a tag here and the copyright class are doing the same thing. so take this approach instead to save code space and allow it to be more readable.
![errorFix2](https://user-images.githubusercontent.com/18339677/109570207-af162500-7ab7-11eb-8223-30592430df5f.jpg)

Please make this change

Reply:
Thanks! Updated the code